Bottom: fdb07e6e2b121ba4ee83390cd267ea9ed0f4b387
Top:    77aedf83fbd17dbb51fde3ef3414da3987cd1096
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-26 19:48:53 +0100

Refresh of redefine-pin-pad-name-label-te

---

diff --git a/src/draw.c b/src/draw.c
index 1315707..7d7a594 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -199,7 +199,8 @@ _draw_pv_name (PinType *pv)
   gui->set_color (Output.fgGC, PCB->PinNameColor);
 
   text.Flags = NoFlags ();
-  text.Scale = 100 * pv->Thickness / MIL_TO_COORD (80);
+  /* Set font height to approx 56% of pin thickness */
+  text.Scale = 56 * pv->Thickness / FONT_CAPHEIGHT;
   text.X = box.X1;
   text.Y = box.Y1;
   text.Direction = vert ? 1 : 0;
@@ -284,7 +285,8 @@ draw_pad_name (PadType *pad)
   gui->set_color (Output.fgGC, PCB->PinNameColor);
 
   text.Flags = NoFlags ();
-  text.Scale = 100 * pad->Thickness / MIL_TO_COORD (50);
+  /* Set font height to approx 90% of pin thickness */
+  text.Scale = 90 * pad->Thickness / FONT_CAPHEIGHT;
   text.X = box.X1;
   text.Y = box.Y1;
   text.Direction = vert ? 1 : 0;
